server/app/genbank.py

```diff
@@ -56,7 +66,7 @@ def my_filter(feature: Any) -> bool:
     return my_filter
 
 
-# Function that reads in a genbank file and converts it into a json
+# Function that reads in a genbank file and converts it into a GenBankData object
```

Comment:
Should be a docstring. Don't need to name types because they're in the annotations.

Reply:
I think I did it already:
```python
def convert_gbk_to_vector(genbank_file, level: VectorLevel) -> GenbankData:
    """
    Function that reads in a genbank file and converts it into a GenBankData object.
    """
```